wca-regulations.md

```diff
@@ -501,6 +501,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
     - E3b) 3x3x3 Cubes (at most 3, self-supplied), as described in [Article 3](regulations:article:3).
     - E3c) Stickers (self-supplied).
     - E3d) Stopwatch or watch (self-supplied) for keeping track of the elapsed time, if it is approved by the WCA Delegate.
+- E4) The Wca Delegate may choose to publish the solutions (e.g. photo of the score sheets or transcriptions of solutions).
```

Comment:
Here's a way to phrase it that is more consistent with other Regulations:

```suggestion
- E4) Competitor solutions (e.g. photo of the score sheets, transcriptions of solutions) may be published at the discretion of the WCA Delegate.
```

(We usually use commas instead of "or", and "WCA in "WCA Delegate" should be uppercase.")

Reply:
@hanwu85, this has a thumbs-up from @Ivan-Brigidano, are you alright with it?

Reply:
@lgarron, Yes, this looks okay to me.

Reply:
Thanks!